build.sbt

```diff
@@ -10,32 +10,32 @@ name := "couchdb-scala"
 
 version := "0.5.3-SNAPSHOT"
 
-scalaVersion := "2.11.5"
+scalaVersion := "2.11.7"
 
 description := "A purely functional Scala client for CouchDB"
 
 homepage := Some(url("https://github.com/beloglazov/couchdb-scala"))
 
 licenses := Seq("The Apache Software License, Version 2.0"
-  -> url("http://www.apache.org/licenses/LICENSE-2.0.txt"))
+                  -> url("http://www.apache.org/licenses/LICENSE-2.0.txt"))
 
 resolvers += "Scalaz Bintray Repo" at "http://dl.bintray.com/scalaz/releases"
 
 libraryDependencies ++= Seq(
-  "org.scalaz"                  %% "scalaz-core"                 % "7.1.0",
-  "org.scalaz"                  %% "scalaz-effect"               % "7.1.0",
-  "org.http4s"                  %% "http4s-core"                 % "0.8.2",
-  "org.http4s"                  %% "http4s-client"               % "0.8.2",
-  "org.http4s"                  %% "http4s-blazeclient"          % "0.8.2",
-  "com.lihaoyi"                 %% "upickle"                     % "0.2.6",
-  "com.github.julien-truffaut"  %% "monocle-core"                % "1.0.1",
-  "com.github.julien-truffaut"  %% "monocle-macro"               % "1.0.1",
-  "org.log4s"                   %% "log4s"                       % "1.1.3",
-  "org.specs2"                  %% "specs2"                      % "2.4.16" % "test",
-  "org.typelevel"               %% "scalaz-specs2"               % "0.3.0"  % "test",
-  "org.scalacheck"              %% "scalacheck"                  % "1.12.1" % "test",
-  "org.scalaz"                  %% "scalaz-scalacheck-binding"   % "7.1.0"  % "test",
-  "ch.qos.logback"              %  "logback-classic"             % "1.1.2"  % "test"
+ "org.scalaz"                  %% "scalaz-core"                 % "7.1.3",
+ "org.scalaz"                  %% "scalaz-effect"               % "7.1.3",
+ "org.http4s"                  %% "http4s-core"                 % "0.9.3",
+ "org.http4s"                  %% "http4s-client"               % "0.9.3",
+ "org.http4s"                  %% "http4s-blaze-client"          % "0.9.3",
+ "com.lihaoyi"                 %% "upickle"                     % "0.3.5",
+ "com.github.julien-truffaut"  %% "monocle-core"                % "1.1.1",
+ "com.github.julien-truffaut"  %% "monocle-macro"               % "1.1.1",
+ "org.log4s"                   %% "log4s"                       % "1.1.3",
+ "org.specs2"                  %% "specs2"                      % "2.4.16" % "test",
+ "org.typelevel"               %% "scalaz-specs2"               % "0.3.0"  % "test",
+ "org.scalacheck"              %% "scalacheck"                  % "1.12.1" % "test",
+ "org.scalaz"                  %% "scalaz-scalacheck-binding"   % "7.1.0"  % "test",
+ "ch.qos.logback"              %  "logback-classic"             % "1.1.2"  % "test"
```

Comment:
Could you please not change the indentation here and below, it screws up the diffs :)